recipes/pytest-tornasync/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytest-tornasync" %}
+{% set version = "0.6.0.post1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5fe41d6d98d4650898c082a69338d25879cadd0e55805c9a02bc3eb7212a2ea1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools
+  run:
+    - pytest >=3.0
+    - python >=3.5
+    - tornado >=5.0
+
+test:
+  imports:
+    - pytest_tornasync
+  requires:
```

Comment:
You don't need this section, those dependencies are already declared in `run`. 👍

Reply:
Nice. I´ll remove them.

Reply:
Done.